content/source/docs/enterprise/private/monitoring.html.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: "enterprise2"
+page_title: "Monitoring Private Terraform Enterprise"
+sidebar_current: "docs-enterprise2-private-installer-monitoring"
+---
+
+# Monitoring Terraform Enterprise Installer Migration
```

Comment:
To follow convention, this title would be something like "Monitoring a Private Terraform Enterprise Installer-based Instance" (or just Instance, if it's not installer specific).

Reply:
I think this was a bad copy-paste :face-palm: -- i'll fix that ASAP